docs/topics/ref_resolved-issues-7-1-1.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+//docs/release_notes-7.1.1/master.adoc
+
+
+:_newdoc-version: 2.18.2
+:_template-generated: 2024-07-01
+
+:_mod-docs-content-type: REFERENCE
+
+[id="resolved-issues-7-1-1_{context}"]
+= Resolved issues
+
+This section provides highlighted issues that have been resolved in {ProductFullName} 7.1.1.
+
+.An application analysis no longer fails when run with the `--bulk` option
+
+Previously, if you ran an application analysis with the `--bulk` option, it failed with the following error:
+----
```

Comment:
should we label what language this error message is in?

Reply:
Let's check with SMEs :-)

Reply:
Resolved with the QE, no need to add anything